### PR TITLE
Committing files after completing the task 100%

### PR DESCRIPTION
**Pull request by vaibhavNagvekarSJI**

### CODE REVIEW

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	public SafetyInformationPage() {
+		PageFactory.initElements(driver, this);
+		actions = new Actions(driver);
+	}
+
+	/**
+	 * Locators
+	 */
+	@FindBy(xpath = "//h1[@id='content-main']")
+	WebElement bannerTitle;
```
Comment (goutamnaik007) [inline BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java]: @vaibhavNagvekarSJI define the locators as private

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java [hunk 2]
```diff
@@ -71,7 +70,20 @@ public void verifyLogoAndTitle(String expectedTitle) {
...
 			Assert.fail("Logo not displayed");
 		}
+	}
 
+	/**
+	 * Function to click on close button of signup pop-up
+	 *
+	 */
+	public void closeSignupPopup() {
+		Action.waitFor(5000);
```
Comment (goutamnaik007) [inline BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java]: @vaibhavNagvekarSJI replace the above method with the one which I have sent you

File: BenadrylAutomation/testng.xml [hunk 3]
```diff
@@ -13,6 +13,8 @@
 			<class name="com.jnj.testcases.ComparePageTest" /> 
 			<class name="com.jnj.testcases.FooterPageTest" />
 			<class name="com.jnj.testcases.OurIngredientsPageTest" />
+			<class name="com.jnj.testcases.SafetyInformationPageTest" />
+
```
Comment (goutamnaik007) [inline BenadrylAutomation/testng.xml]: @vaibhavNagvekarSJI Please remove the blank line

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java (hunk 1 above)
Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java]: fixed

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java (hunk 2 above)
Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java]: fixed

File: BenadrylAutomation/testng.xml (hunk 3 above)
Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/testng.xml]: fixed

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java (hunk 1 above)
Comment (goutamnaik007) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java]: Verified @vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java (hunk 2 above)
Comment (goutamnaik007) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java]: Verified @vaibhavNagvekarSJI

File: BenadrylAutomation/testng.xml (hunk 3 above)
Comment (goutamnaik007) [reply BenadrylAutomation/testng.xml]: Verified @vaibhavNagvekarSJI

**Pull request merged**